pkg/cri/config/config.go

```diff
@@ -88,6 +88,14 @@ type ContainerdConfig struct {
 	DiscardUnpackedLayers bool `toml:"discard_unpacked_layers" json:"discardUnpackedLayers"`
 }
 
+// LogConfig contains toml config related to log
+type LogConfig struct {
+	// LogLimit is the parameter of time.NewLimiter which defines the maximum frequency of writing container logs
```

Comment:
Can we have more detailed description? E.g., "the number of the lines per second"

Reply:
Thanks, I will change the description